lib/config.js

```diff
@@ -435,7 +436,6 @@ function collectDefaultsFromDefinitions(NODE_DEBUG_MODE) {
   });
 
   if (NODE_DEBUG_MODE) {
-    options['web-host'] = '127.0.0.1';
```

Comment:
Please keep `127.0.0.1` for `NODE_DEBUG_MODE`. This mode is triggered when debugging via `node-debug app.js`, in which case the front-end is immediately opened in the browser.